### ISSUE TITLE
attribute using property wrappers

### ISSUE THREAD

**Issue by helje5** [SalehAlbuga/azure-functions-swift#8]
> Unlike C#, Swift does not support Attributes to mark bindings types and directions

You should be able to use the new property wrappers in Swift 5.1 to annotate things.

**Comment by SalehAlbuga** [comment]
Yes I've been experimenting with property wrappers. At the time of the initial development of the worker I used Swift 5.0. Also I've been testing backward compatibility. I think it'll be great to implement property wrappers in the worker, but since it's only in 5.1 and above, I think it'll be good to wait a bit to have more versions supported. But thank you for pointing this out, I'm updating the docs to make it clearer :)
